simpleast-core/src/main/java/com/discord/simpleast/code/CodeRules.kt

```diff
@@ -172,6 +172,23 @@ object CodeRules {
         PATTERN_TEXT.toMatchGroupRule(),
     )
 
+    val queryLangauge = listOf<Rule<R, Node<R>, S>>(
+        createSingleLineCommentPattern("#")
+            .toMatchGroupRule(stylesProvider = codeStyleProviders.commentStyleProvider),
+        Pattern.compile("""^"[\s\S]*?(?<!\\)"(?=\W|\s|$)""")
+            .toMatchGroupRule(stylesProvider = codeStyleProviders.literalStyleProvider),
+        createWordPattern("true|false|null").pattern().toPattern(Pattern.CASE_INSENSITIVE)
```

Comment:
this is a small hack so I can reuse the `createWordPattern` regex but add an option.
it only happens on creation so should be low cost overall.